src/tools/shmmon/eds/edid.h

```diff
@@ -406,8 +406,8 @@ static inline void
 edid_manufacturer(const struct edid * const edid, char manufacturer[4])
 {
     manufacturer[0] = '@' + ((edid->manufacturer & 0x007c) >> 2);
-    manufacturer[1] = '@' + (((edid->manufacturer & 0x0003) >> 00) << 3)
-                          | (((edid->manufacturer & 0xe000) >> 13) << 0);
+    manufacturer[1] = '@' + ((((edid->manufacturer & 0x0003) >> 00) << 3)
+                             | (((edid->manufacturer & 0xe000) >> 13) << 0));
```

Comment:
doh. I actually got this wrong (in C, `+` has higher precedence than `|`). Which was intended here?

Reply:
not sure, that one comes from in the drm-code, either kernelside or userside.

Reply:
Looks like you copied this from https://github.com/compnerd/eds/blob/d239028cf9152e1165f816bcc29c6de12b989258/src/eds/edid.h#L403-L411 ?

From wikipedia:

 > Manufacturer ID. This is a legacy Plug and Play ID assigned by Microsoft, which is a big-endian 16-bit value made up of three 5-bit letters: 00001, A; 00010,  B; …; 11010, Z.  E.g., 24 4d, 0 01001 00010 01101, "IBM".
>  - Bit 15 | Reserved, always 0
>  - Bits 14–10 | First letter of manufacturer ID (byte 8, bits 6–2)
>  - Bits 9–5 | Second letter of manufacturer ID (byte 8, bit 1 through byte 9 bit 5)
>  - Bits 4–0 | Third letter of manufacturer ID (byte 9 bits 4–0)

So really `|` vs `+` doesn't even matter here: `@` (ascii 64) doesn't have low bits set.